### PR TITLE
fix mobile contacts list

### LINKED ISSUE #2289: Contacts list scroll broken on mobile with contacts v4
### Describe the bug
Contacts are not displayed in mobile view, only on Desktop
......

**To Reproduce**
I have over 700 contacts. These are displayed on the desktop without any problems with the current beta version. But no contacts are displayed on the smartphone.

This problem did not occur with version 3.x.

See screenshots....
Screenshot_20210618-093358_Chrome
InkedScreenshot

### Server configuration

**Nextcloud version:** 21.0.2

**Contacts version:** 4.0.0.-Beta 2 (Circle is NOT installed)

**Updated from an older Nextcloud or fresh install:** updated from NC 15 an so on....

### Client configuration
**Browser:**
DESKTOP: Windows 10+ Firefox 89.0.1 
MOBILE: Android+Samsung Browser (V.14.01) or Chrome (same same)

#### Browser log
No Browser-Log on my Android-Device, sorry

### PR DESCRIPTION
**Pull request by szaimen**
Fix #2289

Signed-off-by: szaimen <szaimen@e.mail.de>

### CODE REVIEW

File: src/components/ContactsList.vue [hunk 1]
```diff
@@ -154,7 +154,7 @@ export default {
 <style lang="scss" scoped>
 // Make virtual scroller scrollable
 .contacts-list {
-	max-height: 100%;
+	max-height: calc(100vh - 50px);
```
Comment (skjnldsv) [inline src/components/ContactsList.vue:157]: https://github.com/nextcloud/server/blob/5fd9ce9e8de61d606cb53eaf24a1c51f2fb2b87a/core/css/css-variables.scss#L59

**Review by skjnldsv — CHANGES_REQUESTED** [review]
See comment

File: src/components/ContactsList.vue (hunk 1 above)
Comment (szaimen) [reply src/components/ContactsList.vue]: done

**Comment by szaimen** [comment]
> See comment

done

**Review by skjnldsv — APPROVED** [review]

**Comment by skjnldsv** [comment]
Didn't test

**Comment by szaimen** [comment]
Just tested again and works

**Pull request merged**